src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -1454,68 +1454,64 @@ void ModControllableAudio::receivedCCFromMidiFollow(ModelStack* modelStack, Clip
 					    ccNumber, midiEngine.midiFollowDisplayParam);
 					// check if model stack is valid
 					if (modelStackWithParam && modelStackWithParam->autoParam) {
-						if (modelStackWithParam->getTimelineCounter()
-						    == view.activeModControllableModelStack.getTimelineCounterAllowNull()) {
-
-							// get current value
-							int32_t oldValue =
-							    modelStackWithParam->autoParam->getValuePossiblyAtPos(view.modPos, modelStackWithParam);
-
-							// convert current value to knobPos to compare to cc value being received
-							int32_t knobPos = modelStackWithParam->paramCollection->paramValueToKnobPos(
-							    oldValue, modelStackWithParam);
-
-							// add 64 to internal knobPos to compare to midi cc value received
-							// if internal pos + 64 is greater than 127 (e.g. 128), adjust it to 127
-							// because midi can only send a max midi value of 127
-							int32_t knobPosForMidiValueComparison = knobPos + kKnobPosOffset;
-							if (knobPosForMidiValueComparison > kMaxMIDIValue) {
-								knobPosForMidiValueComparison = kMaxMIDIValue;
-							}
+						// get current value
```

Comment:
why is this being changed? With the indentation change I'm not sure what the actual difference is

Reply:
it's cause I removed this - this code basically checks that you're trying to control the same thing as the gold knobs. But since we're allowing controlling the last selected clip, this won't work anymore and isn't needed as we are controlling something different than the gold knobs.

```
if (modelStackWithParam->getTimelineCounter()
						    == view.activeModControllableModelStack.getTimelineCounterAllowNull()) {
```